File: 2023/0-str-230417_0512/1316-BAD.cpp
#include <cstdio>
#include <cstring>

int N, ret;
char s[101];
bool visit[26];

int main()
{
    scanf("%d", &N);
    while (N--) {
        scanf("%s", s);

        memset(visit, 0, 26);
        
        int inc = 1;
        char prev = -1;
        for (int i = 0; s[i]; i++) {
            if (prev == s[i])   continue;
            prev = s[i];

            if (visit[s[i] - 'a']) {
                inc = 0;
                break;
            }
            visit[s[i] - 'a'] = true;
        }
        ret += inc;
    }
    printf("%d", ret);
}

File: boj/2023/0-str-230417_0512/1316-BAD.py
N = int(input())

ret = 0
for _ in range(N):
    inc = 1

    visit = set()
    prev = -1
    for c in input():
        if c == prev:   continue
        prev = c

        if c in visit:
            inc = 0
            break
        visit.add(c)
    
    ret += inc

print(ret)